src/hotspot/share/runtime/interfaceSupport.inline.hpp

```diff
@@ -350,13 +351,11 @@ class RuntimeHistogramElement : public HistogramElement {
 
 #define VM_LEAF_BASE(result_type, header)                            \
   TRACE_CALL(result_type, header)                                    \
-  debug_only(NoHandleMark __hm;)                                     \
```

Comment:
Shouldn't `NoSafepointVerifier` be added here and then it can be removed from `JRT_LEAF`?